effectzgpt/backend/app/engine/node_postprocessors.py

```diff
@@ -0,0 +1,7 @@
+from llama_index.core.postprocessor import MetadataReplacementPostProcessor
+from app.engine.constants import DEFAULT_META_DATA
```

Comment:
DEFAULT_META_DATA is not defined in constants.py

Reply:
@iujinasena this has been fixed.